Sming/Wiring/WString.cpp

```diff
@@ -17,7 +17,14 @@
 */
 
 #include "WiringFrameworkIncludes.h"
-#include <stdlib.h>
+
+/*
+ * Want memmem() to replace use of strstr() so we can search on byte strings.
+ * This is a GNU extension so not available with -std=c11
```

Comment:
I would prefer if we keep to standard c11 instead of gnu c11. One possible scenario is that other party might release much better C/C++ compiler based on [Clang](http://clang.llvm.org/). Isn't there a compiler independent solution for this?

Reply:
Both memmem() and strcasecmp() probably belong in system/stringutil. I can do that and commit to this PR if you are agreeable.

Reply:
Sounds reasonable to me. @mikee47 make that change so that I can test that PR on my device(s) and merge it, if nothing pops-up.